template.distillery/Makefile.style

```diff
@@ -67,7 +83,7 @@ $(CSSDIR):
 $(CSS_DEST): $(LOCAL_CSS) | $(CSSDIR) check_sed
 	HASH=`cat $< | md5sum | cut -d ' ' -f 1` && \
 	cp $< $(CSS_PREFIX)_$$HASH.css && \
-	$(SED) '1s/^/@charset "UTF-8";/' $(CSS_PREFIX)_$$HASH.css && \
+        $(SED) '1s/^/@charset "UTF-8";/' $(CSS_PREFIX)_$$HASH.css && \
```

Comment:
You replace tabs by more spaces. I think it would be better to replace it by tabs to keep the same styling code.